IBAnimatable/Animatable.swift

```diff
@@ -76,25 +76,25 @@ public extension Animatable where Self: UIView {
   public func animate(_ animation: AnimationType? = nil, completion: AnimatableCompletion? = nil) {
     switch animation ?? animationType {
     case let .slide(way, direction):
-      slide(way: way ?? .in, direction: direction ?? .right, completion: completion)
+      slide(way, direction: direction, completion: completion)
```

Comment:
Actually those default value were useless. Beta 6 now put a warning when `??` operator is used with non-optional values. 
We decided some times ago to make `AnimationType` enum parameters non-optional. 
So we don't have anymore defaultValue when called from code, Just the documentation to indicate the default value. DefaultValues are still applied when called from IB (in `init(string: String?)`